.pre-commit-config.yaml

```
@@ -143,9 +143,11 @@ repos:
     hooks:
       - id: verify-copyright
         exclude: |
-          (?x)
-              cpp/include/cudf_test/cxxopts[.]hpp$
-
+          (?x)^(
+            cpp/include/cudf_test/cxxopts[.]hpp|
+            cpp/src/io/parquet/ipc/Message_generated[.]h|
+            cpp/src/io/parquet/ipc/Schema_generated[.]h
+          )$
```

You don't want the `$` at the end of the pattern because it would prevent us from matching certain regexes. Instead, use `$` in each sub-pattern.
```suggestion
          (?x)^(
            cpp/include/cudf_test/cxxopts[.]hpp$|
            cpp/src/io/parquet/ipc/Message_generated[.]h$|
            cpp/src/io/parquet/ipc/Schema_generated[.]h$
          )
```